yt_dlp/extractor/facebook.py

```diff
@@ -678,6 +678,13 @@ def parse_attachment(attachment, key='media'):
                 # honor precise duration in video info
                 if video_info.get('duration'):
                     webpage_info['duration'] = video_info['duration']
+                # preserve preferred_thumbnail in video info
```

Comment:
(I edited the title to remove the misleading word 'preferred')

From my observation, thumbnailImage and preferred_thumbnail are the same image although the urls are different in parameter (_nc_sid) when both of them are provided. (something FB gives us only one of them)
Therefore, the order here does not matter.

or, have you found them different?

Reply:
> have you found them different?

No, I just thought since you specifically referenced `preferred_thumbnail` that it would be the, uh, preferred thumbnail. But if you say it doesn't matter then leaving it as-is is fine IMO